.github/workflows/bc-lint.yml

```
@@ -0,0 +1,28 @@
+name: BC Lint
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+    # branches-ignore:
+    #   - <branches to ignore bc-linter>
+    paths:
+      - vllm/attetion/**
+      - vllm/core/**
+      - vllm/v1/attetion/**
+      - vllm/v1/core/**
+
+jobs:
+  bc_lint:
```

You want this line https://github.com/pytorch/pytorch/blob/main/.github/workflows/lint-bc.yml#L19C5-L19C45 to run this job only on PR submitted to vllm

```suggestion
  bc_lint:
    if: github.repository_owner == 'vllm-project/'
```

---

in fact we need this in all of our actions 😆